modules/templates-cli-generating-list-of-objects.adoc

```diff
@@ -57,9 +50,7 @@ $ oc process -f my-rails-postgresql \
     -p POSTGRESQL_DATABASE=mydatabase
 ----
 +
-.. The JSON file can either be redirected to a file or applied directly without
-uploading the template by piping the processed output to the `oc create`
-command:
+.. The JSON file can either be redirected to a file or applied directly without uploading the template by piping the processed output to the `oc create` command:
```

Comment:
This is a holdover from earlier versions, I think, and today it almost certainly outputs YAML.